### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJEclipse_Plugins_Screen Reader: JAWS is not reading the name of the search result list item

### ISSUE THREAD

**Issue by soumyatippapuram** [microsoft/azure-tools-for-java#3386]
[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier!

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #JAWS 

Environment Details:
Download and Install IntelliJ IDEA from https://docs.microsoft.com/en-us/java/azure/intellij/azure-toolkit-for-intellij-create-hello-world-web-app?view=azure-java-stable
IntelliJ IDEA 2019.1.3
Build #IU-191.7479.19

OS Details: ​
OS: Windows (Version 1903, Build: 18362.175)
MAS Violated: MAS 1.3.1

Repro Steps:
1) Start JAWS and open IntelliJ IDEA application
2) Press 'Ctrl+Alt+S' to open application settings/preferences popup
3) Tab till 'Plugins' tab and press enter
4) Tab till search box and type "Azure"
5) Check if the JAWS is reading the search results properly

Actual Result:
JAWS is not reading the complete name of the result list item. It is just reading the list items as "Azure"

Expected Result:
JAWS should read the full name for the search results, i.e., it should read as "Azure Toolkit for IntelliJ"

User Impact:
Screen Reader users will not know the which option to choose if the name is not read properly

MAS Reference:
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=35e0fa4e-05f4-483e-98b5-2ddae34a2404)

[Bug_3386_IntelliJ_Plugins_JAWS.pptx](https://github.com/microsoft/azure-tools-for-java/files/3466895/Bug_3386_IntelliJ_Plugins_JAWS.pptx)

**Comment by andxu** [comment]
@soumyatippapuram The behavior of searching in marketplace is controlled by IntelliJ, which is out of the scope of our toolkit, if you download IntelliJ, have a fresh install of it and press 'Ctrl+Alt+S' to open application settings/preferences popup, tab till 'Plugins' tab and press enter ANYTHING not related to azure, the search results can not be read by JAWS.

**Comment by andxu** [comment]
Importance of experience: Little, it is a nice to have feature reading the search results once user has finished the typing.
Quality of experience: not impacted. Since you can navigate using tab to the search list and JAWS will start reading the result
Alternate solutions: Use tabs to navigate through result list

**Comment by Flanker32** [comment]
@soumyatippapuram This issue is an external issue which has been linked to AAT, please help us review it, thanks!

**Comment by soumyatippapuram** [comment]
Reported this issue to IntelliJ (https://youtrack.jetbrains.com/issue/IDEA-222344)

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and this is non blocking scenario

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749776
